src/shell/app-shell.html

```diff
@@ -92,10 +92,6 @@
 
       attached() {
         this.loadLocales(this.is);
-      },
-
-      _pageChanged(page, oldPage) {
```

Comment:
This is causing the longer to fail as long as it's there and the content of it isn't...